# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x86/memtest86/ld-gnu-linkonce.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- memtest86+_v1.11/memtest_shared.lds	2003-12-30 08:20:16.000000000 +0100
+++ memtest86+_v1.11-jsaw/memtest_shared.lds	2004-02-23 19:38:22.000000000 +0100
@@ -9,6 +9,7 @@
 		*(.text)
 		*(.text.*)
 		*(.plt)
+		*(.gnu.linkonce.*)
 		_etext = . ;
 	} = 0x9090
 	.rodata : {
